### PR TITLE
Embeddings executor.

### PR DESCRIPTION
**Pull request by fissoreg**
`Executor` class to compute embeddings with `ProtBERT`.

### CODE REVIEW

File: backend/my_executors.py [hunk 1]
```diff
@@ -1,41 +1,53 @@
...
         with torch.no_grad():
             outputs = self.model(**encoded_inputs)
             embeds = outputs.last_hidden_state[:, 0, :].detach().numpy()
             for doc, embed in zip(docs, embeds):
                 doc.embedding = embed
 
         return docs
-#%%
+
+    def preprocessing(self, sequences: Sequence[str]) -> List[str]:
```
Comment (georgeamccarthy) [inline backend/my_executors.py:38]: This notation is new to me!

**Comment by georgeamccarthy** [comment]
Looks good

File: backend/my_executors.py (hunk 1 above)
Comment (fissoreg) [reply backend/my_executors.py:38]: You mean the type annotations? It's a relatively new features of Python, and I have noticed that in the Jina source code they use annotated code. So I aligned to the style :)

Comment (georgeamccarthy) [reply backend/my_executors.py:38]: Particularly the arrow notation (-> List[str]) after the parameters. [So it specifies the return type. ](https://medium.com/@thomas_k_r/whats-this-weird-arrow-notation-in-python-53d9e293113) Cool!

**Pull request merged**